planeshifted/planeshifted-guide/phyrexia/race-compleated.xml

```diff
@@ -0,0 +1,174 @@
+		<supports>Phyrexian Design</supports>
+		<description>
+			<p>While you aren't wearing armor, your base Armor Class is 12 + your Constitution modifier.</p>
+		</description>
+		<sheet>
+			<description>While you aren't wearing armor, your Armor Class is {{ac:natural armor:compleated}}.</description>
+		</sheet>
+		<rules>
+			<stat name="ac:natural armor:compleated" value="12" bonus="base" />
+			<stat name="ac:natural armor:compleated" value="constitution:modifier" bonus="ability" />
+			<stat name="ac:calculation" value="ac:natural armor:compleated" bonus="calculation" alt="Natural Armor (Compleated)" />
+		</rules>
+	</element>
+
+	<!-- Glistening Oil Hazard -->
+	<element name="Glistening Oil" type="Item" source="Planeshifted Guide to Phyrexia" id="ID_PHYREXIA_RACIAL_TRAIT_GLISTENING_OIL">
+		<description>
+			<p><i>Hazard</i></p>
+			<p>Glistening oil is an infectious black ichor. Some Phyrexian creatures weaponize this oil by coating their weapons with it. On New Phyrexia, the oil can be found leaking from structures, collecting in pools and fountains, or dripping from the spheres above. A pool of glistening oil typically covers a 10-foot square. A creature can avoid being struck by dripping glistening oil with a successful DC 10 Dexterity saving throw. A creature that comes into contact with glistening oil must make a DC 15 Constitution saving throw or become afflicted with phyresis until cured. Phyresis can't be cured by any means short of a <i>wish</i> spell.</p>
+			<p class="indent">At the end of each long rest, a creature afflicted with phyresis must repeat the Constitution saving throw. A creature wearing hexgold armor has advantage on this saving throw. A creature under the effects of a potion of halo* or a cosmos elixir* automatically succeeds on this saving throw. If a creature fails its saves ten times (the failures don't need to be consecutive), the affliction ends, and creature gains the following traits:</p>
+			<p class="indent"><strong>All Will Be One.</strong> You gain a new character bond that supersedes all others: "I serve Phyrexia." You are free to interpret this bond in accordance with your alignment; you may seek to hide this new bond from others to further Phyrexia's goals in secret, or you may attempt to infect others by force.</p>
+			<p class="indent"><strong>Glistening Strike.</strong> Your unarmed strikes are coated with glistening oil.</p>
+			<p class="indent"><strong>Language.</strong> You can speak, read, and write Phyrexian.</p>
+		</description>
+		<setters>
+			<set name="category">Additional Feature</set>
+			<set name="slot">misc</set>
+			<set name="inventory-hidden">false</set>
+		</setters>
+		<sheet display="false" />
+		<rules>
+			<grant type="Racial Trait" id="ID_PHYREXIA_RACIAL_TRAIT_GLISTENING_OIL_ALL_WILL_BE_ONE" />
+			<grant type="Racial Trait" id="ID_PHYREXIA_RACIAL_TRAIT_GLISTENING_OIL_STRIKE" />
+			<grant type="Language" id="ID_PSLE_PSGP_LANGUAGE_PHYREXIA" />
+		</rules>
```

Comment:
These should be granted by the race itself. Granting an item in a race doesn't work.
Remember to apply TCoE rules to the language as well.